### PR TITLE
157247809 Fix scheduled task time bug

### PR DESCRIPTION
**Pull request by tatut**
# Fixed
- 1st scheduled task time was skipped if restarting before it

### CODE REVIEW

File: ote/src/clj/ote/tasks/pre_notices.clj [hunk 1]
```diff
@@ -130,22 +130,18 @@
              (log/warn "Error while sending a notification" e))))))))
 
 
-(defrecord PreNoticesTasks [at email-config interval]
+(defrecord PreNoticesTasks [email-config]
   component/Lifecycle
   (start [{db :db :as this}]
     (assoc this
-      ::stop-tasks [(chime-at (drop 1 (periodic-seq at (t/seconds
-                                                         (hours->seconds (or interval
```
Comment (K1ll3rF0x) [inline ote/src/clj/ote/tasks/pre_notices.clj]: Why did you remove the usage of interval option? It is much easier to change task intervals from config.edn than in code, no new build is needed.

Comment (tatut) [reply ote/src/clj/ote/tasks/pre_notices.clj]: I think we don't need it, it is even easier to call the timed function from the RPEL for testing purposes.

The `daily-at` gives a sequence of daily times (without an interval).

Comment (K1ll3rF0x) [reply ote/src/clj/ote/tasks/pre_notices.clj]: Ok, could you also remove the interval settings from config.end and aws config.edn templates?
